index.js

```diff
@@ -187,7 +196,7 @@ const checkArrows = (actionCwd) => {
 
 // Status line
 exports.decorateHyper = (Hyper, { React }) => {
-    return class extends React.Component {
+    return class extends React.PureComponent {
```

Comment:
I was logging things locally and noticed that the React component would render every 100ms (the `setInterval()` interval). Changing this to a [`React.PureComponent`](https://facebook.github.io/react/docs/react-api.html#react.purecomponent) will cause the component to render only when the state has changed instead of every 100ms. 👍